contrib/cirrus/add_second_partition.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# N/B: This script could mega f*!@up your disks if run by mistake.
+#      it is left without the execute-bit on purpose!
+
+# $SLASH_DEVICE is the disk device to be f*xtuP
+SLASH_DEVICE="/dev/sda"  # Always the case on GCP VM Images
+
+# The unallocated space results from the difference in disk-size between VM Image
+# and runtime request.  The check_image.sh test includes a minimum-space check,
+# with the Image size set initially lower by contrib/cirrus/packer/libpod_images.yml
+NEW_PART_START="50%"
+NEW_PART_END="100%"
+
+set -eo pipefail
+
+source $(dirname $0)/lib.sh
+
+# Protection against double-execution
+if [[ ! -r "/root" ]] || [[ -r "/root/second_partition_ready" ]]
+then
+    echo "Warning: Ignoring attempted execution of $(basename $0)"
+    exit 0
+fi
+
+[[ -n "type -P parted" ]] || \
```

Comment:
This is not what you meant to do.

Reply:
whoa! good catch!

Reply:
okay, fixed (and did an executable-test instead of string length)